protocol-units/da/m1/light-node-client/src/test/e2e/raw/sequencer.rs

```diff
@@ -1,5 +1,5 @@
 use crate::*;
-use movement_types::Block;
+use movement_types::block::Block;
```

Comment:
The "main type" in the module, and any other unambiguously named types, could be re-exported to the top level.